tools/downloader/README.md

```diff
@@ -379,6 +381,70 @@ Toolkit will still be created, so that you can inspect it.
 See the "Shared options" section for information on other options accepted by
 the script.
 
+## Model Compiler usage
+
+The basic usage is to run the script like this:
+
+```sh
+./compiler.py --all
+```
+
+This will compile all models for which it is supported. Other models
+are ignored.
+
+The `--all` option can be replaced with other filter options to compile only
+a subset of models. See the "Shared options" section.
+
+The current directory must be the root of a tree of model files created by the model
+converter. To specify a different model tree path, use the `--model_dir` option:
+
+```sh
+./compiler.py --all --model_dir my/model/directory
+```
+
+By default, the compiled models are placed into the same model tree. To place them
+into a different directory tree, use the `-o`/`--output_dir` option:
+
+```sh
+./compiler.py --all --output_dir my/output/directory
+```
+
+By default, the script will compile models in FP16 and FP32 precisions.
+To only produce models in a specific precision, use
+the `--precisions` option:
+
+```sh
+./compiler.py --all --precisions=FP32
+```
+
+The script will attempt to locate Compile Tool using
+the environment variables set by the OpenVINO&trade; toolkit's `setupvars.sh`/`setupvars.bat`
+script. You can override this heuristic with the `--compiler` option:
+
+```sh
+./compiler.py --all --compiler my/openvino/path/compile_tool/compile_tool
+```
+
+It's possible to specify a target device for Compile Tool
+to compile for, by using the `--target_device` option:
+
+```sh
+./compiler.py --all --target_device MYRIAD
+```
+
+The supported values are those accepted by the `-d` option in
+Compile Tool documentation. If this option is unspecified, the default is `MYRIAD`.
```

Comment:
MYRIAD plugin is close to EOL. Do we really need this feature?

I against of merging this.

Reply:
MYRIAD was used just for testing, thats why i set there such option as.  What about VPU?

Reply:
In general I want customers to use common LoadNetwork flow with cache feature enabled. 
Even for VPUX we quite heavy compilation we can use CVS-56054 feature as a solution

CC @myshevts

Reply:
@ilya-lavrenov feature you mention targeted for 2022.1 release, is not it?

Reply:
yes

Reply:
FWIW, I'm inclined to agree with @ilya-lavrenov here. If I understand correctly:

1. Automatic compilation (i.e. via `LoadNetwork`) will become viable in 2022.1 (due to the addition of caching), at which point this tool will no longer be useful.
2. Even while manual compilation is required, the benefit provided by this tool is fairly minimal, because all models are compiled the same way (the only model-specific setting is whether it can be compiled at all).